cpp/arcticdb/processing/clause.hpp

```diff
@@ -614,8 +616,8 @@ struct MergeClause {
 
     [[nodiscard]] std::vector<std::vector<size_t>> structure_for_processing(
             std::vector<RangesAndKey>& ranges_and_keys,
-            size_t start_from) {
-        return structure_by_row_slice(ranges_and_keys, start_from);
+            size_t) {
+        return structure_all_together(ranges_and_keys);
```

Comment:
Do we need this? `structure_for_processing` is only called on the first clause in the pipeline, which should never be `MergeClause`?

Reply:
It is conceivable that we might want to take a bunch of sorted things and merge them all together